crates/runtime/src/libcalls.rs

```diff
@@ -203,6 +206,7 @@ fn memory32_grow(
 }
 
 // Implementation of `table.grow`.
+#[wasmtime_versioned_export_macros::versioned_export(aliases = [table_grow_func_ref, table_grow_externref])]
```

Comment:
I think this (and others on the top-level functions in this file) can be removed because these symbols aren't exported from the wasmtime library

Reply:
I also think this may be causing the segfault on CI since the "mangled" symbols here should be the inline-assembly-generated trampolines, not the Rust-defined functions  (e.g. the functions generated in the `wasm_to_libcall_trampoline` macro invocation)

Reply:
Sounds good, I removed these and instead did the renaming inside of the `wasmtime_asm_macros::asm_func!` calls